firewall/interception/windowskext/handler.go

```diff
@@ -43,16 +48,40 @@ type VerdictRequest struct {
 	packetSize uint32
 }
 
+// Do not change the order of the members! The structure to communicate with the kernel extension.
```

Comment:
I think there are some words missing here. Is suggest: `The structure is used to communicate` ...